CHANGELOG.md

```diff
@@ -20,6 +20,18 @@ Code v99.99.999
 
 -->
 
+## [4.7.1](https://github.com/coder/code-server/releases/tag/v4.7.1) - 2022-09-30
+
+Code v1.71.2
+
+### Changed
+
+- Updated Code to 1.71.2
+
+### Fixed
+
+- Added RPM upgrade flag to install script
```

Comment:
We could expand on this a bit:

```suggestion
- Fix the install script not upgrading code-server when already installed on RPM-based machines
```

Reply:
Oh also that sudo fix got in with the install script too right?

Reply:
`Fix install script installing with su on FreeBSD` or something like that.

Reply:
Good call! I'll update this now.